Annotation/declarative_golden_set.json

```diff
@@ -1575,7 +1709,7 @@
                     }
                 ],
                 "type": "declarative",
-                "clause": "",
+                "clause": "he was puzzled by why he was uncompetitive on the second day of Valencia's post-season MotoGP test on Wednesday having topped the times on day one",
                 "embedded clauses": []

```

Comment:
Is "puzzled by why he was uncompetitive on the second day of xx" not a nested EC?